api/v1alpha1/nicclusterpolicy_types.go

```diff
@@ -288,6 +288,43 @@ type DOCATelemetryServiceSpec struct {
 	Config *DOCATelemetryServiceConfig `json:"config"`
 }
 
+// NicFirmwareStorageSpec contains configuration for the NIC firmware storage
+type NicFirmwareStorageSpec struct {
+	// Create specifies whether to create a new PVC or use an existing one
+	// +kubebuilder:default:=true
+	Create bool `json:"create,omitempty"`
+	// PVCName is the name of the PVC to mount as NIC Firmware storage. Default value: "nic-fw-storage-pvc"
+	// +kubebuilder:validation:MinLength=1
+	// +kubebuilder:validation:MaxLength=253
```

Comment:
Can we create PVC name longer than 63 symbols? This [1] mention 63 symbols limit

[1] https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names

Reply:
Changed maxLength to 63